api/v1alpha1/profilecatalogsource_types.go

```diff
@@ -31,6 +31,22 @@ type ProfileDescription struct {
 	Name string `json:"name,omitempty"`
 	// Profile description
 	Description string `json:"description,omitempty"`
+	// Version
+	// +optional
+	Version string `json:"version,omitempty"`
+	// Catalog is the name of the catalog the profile is listed in
+	// TODO necessary?
+	// +optional
+	Catalog string `json:"catalog,omitempty"`
```

Comment:
We can work this out from the CatalogSource?

Reply:
oh yeh we can get from the metadata, totally meant to do that!

Reply:
as noted out of band - i am going to do this in a separate PR as it has revealed some weirdness with adding more than one catalog, and i want to unblock other work

Reply:
nvm did it anyway